.gitignore

```diff
@@ -6,6 +6,7 @@ node_modules
 test-results
 __diff_output__
 .jest-cache
+__tests__/src/__image_snapshots__
```

Comment:
Let's not add this to `.gitignore` and instead take care of cleaning up the file system after the tests that generate this. That way in the future if we add image snapshots we want to keep they are not ignored.